public/app/plugins/datasource/cloudwatch/query-runner/CloudWatchLogsQueryRunner.ts

```diff
@@ -252,7 +252,7 @@ export class CloudWatchLogsQueryRunner extends CloudWatchRequest {
       this.logQueries[param.refId] = {
         id: param.queryId,
         region: param.region,
-        statsQuery: (param.statsGroups?.length ?? 0) > 0 ?? false,
+        statsQuery: (param.statsGroups?.length ?? 0) > 0,
```

Comment:
Is this related to the bug fix? What is the context for this change?

Reply:
This was a quick fix for my linter that didn't seem big enough to warrant a separate pr (we never hit the `?? false` portion). I can remove it if we want.